test/contrib/rails/apps/cache.rb

```diff
@@ -0,0 +1,23 @@
+module RedisStoreDataAccessor
+  attr_accessor :data
+end
+
+def get_cache
+  # Rails3 does not define data as an accessor, we need it for our tests
+  if defined?(::ActiveSupport::Cache::RedisStore)
+    unless ::ActiveSupport::Cache::RedisStore.method_defined?(:data)
+      ::ActiveSupport::Cache::RedisStore.prepend RedisStoreDataAccessor
+    end
+  end
+
+  cache = nil
+  redis_host = ENV['DATADOG_TEST_REDIS_CACHE_HOST']
+  redis_port = ENV['DATADOG_TEST_REDIS_CACHE_PORT']
+  if redis_host && redis_port
+    puts "using redis cache on #{redis_host}:#{redis_port}"
+    cache = :redis_store, { host: redis_host, port: redis_port }
```

Comment:
instead of using different ``ENV``, I think that using a plain:
```
config.cache_store = :redis_store if ENV['REDIS_URL']
```
is enough. If I'm not wrong, to be 12-factor app compliant gems are looking for this kind of env var (I may be wrong!). Anyway, all other changes to switch the cache are OK.